app/views/sorns/search.html.erb

```diff
@@ -1,4 +1,13 @@
-  <div class="grid-container margin-top-4">
+<header id="search-header" class="margin-top-2">
+  <div class="grid-container">
+    <div class="grid-row grid-gap grid-col-10">
```

Comment:
Seeing it in Source Sans, the measure is a little long. Could we make it `grid-col-9`?